sympy/physics/control/lti.py

```diff
@@ -249,6 +261,102 @@ def expand(self):
         """
         return TransferFunction(expand(self.num), expand(self.den), self.var)
 
+    def dcgain(self):
```

Comment:
`dc_gain` seems like a more pythonic name to me, but maybe it's not sympythonic. I'll let some other sympy maintainer decide.

Reply:
Yeah, right... How about `low_frequency_gain`? Ping @moorepants @ilayn

Reply:
The convention is settled on this pretty much. I think `dc_gain` is the one that most would expect.

Reply:
Looks terse but let's go with it.